### ISSUE TITLE
The challenges document has some value judgments about WCAG 

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wcag#965]
The document  about challenges for large organizations contains some language that is value based and could be used against WCAG in legal proceedings.
https://w3c.github.io/wcag/conformance-challenges/

I've copied the text into a Google doc and made some comments.
https://docs.google.com/document/d/1lUTCNGbP-afvym_6XQkjT6ey_QgtfTATpi2Wv8ZuKlQ/edit?usp=sharing

**Comment by sajkaj** [comment]
David: I'm closing this issue only because it's insufficiently specific to be easily actionable. I recall you mentioning the "too many SC" language during the telecon. That language is no longer in the Challenge #1 text of the [Editor's Draft](https://w3c.github.io/wcag/conformance-challenges/). Can you please provide a specific list of your concerns based on that draft? Thanks!
